doc/src/manual/style-guide.md

```diff
@@ -447,3 +447,43 @@ julia> h(1)
 
 Thus, use `Int` literals when possible, with `Rational{Int}` for literal non-integer numbers,
 in order to make it easier to use your code.
+
+## Workarounds for named positional arguments
+Named positional arguments are not supported in Julia, however, some users may still wish to use a named positional arguments code-style.
+There are several workarounds that can provide an experience similar to named positionals.
+Consider the `mod` method, which a user might call with the ambiguous variables `a` and `b` in the positional arguments:
+```julia
+a = 5; b = 2
+mod( a, b )
+```
+
+One approach is to use inline comment blocks when calling the method:
+```julia
+mod( #=x=# a, #=y=# b )
+```
+
+or with end-of-line comments
+
+```julia
+mod( a, # x
+     b, # y
+    )
+```
+
+Furthermore the use of comments permit more descriptive names while not interfering with Julia's multiple dispatch on positional arguments:
```

Comment:
remove the opinionated "more descriptive"

Reply:
I think within the context of a style guide, and within the specific context of that example, that `more descriptive` is suitable.  I had originally written "... permit the usage of names that the user determines to be more descriptive to themselves or to future readers..." which I felt ended up being word-salad.  I'm trying to get the point across that if the user doesn't find `z=mod(x,y)` to be descriptive to them, but they do find `remainder=mod( dividend, divisor )` to be more descriptive _to them_ that this is an option.

I would also argue that it objectively is "more descriptive", just as `num_multiples=mod( divisor, dividend )` would be objectively less descriptive, not just an opinionated statement.  But if you think I should be more clear with what I intend by "more descriptive" to make it feel less opinionated I can work on my original word-salad to clear it up and find a happy-medium.

Reply:
well, I might call it "objectively more cluttered" :) 

that's the problem with normative statements presented in such a general context

Reply:
Fair enough - I've updated to the following - the **bold** portion is the change:

> Furthermore the use of comments permit **you to insert hints describing a positional argument,** while not interfering with Julia's multiple dispatch on positional arguments: